swagger_django_generator/templates/stubs.py

```diff
@@ -39,7 +39,7 @@ class MockedStubClass(AbstractStubClass):
     {% for verb, info in verbs|dictsort(true) %}
 
     @staticmethod
-    def {{ info.operation }}(request, {% if info.body %}body, {% endif %}{% for ra in info.required_args %}{{ ra.name }}, {% endfor %}{% for oa in info.optional_args %}{{ oa.name }}=None, {% endfor %}*args, **kwargs):
+    def {{ info.operation }}(request, {% if info.body %}body, {% endif %}{% if info.formdata %}formData, {% endif %}{% for ra in info.required_args %}{{ ra.name }}, {% endfor %}{% for oa in info.optional_args %}{{ oa.name }}=None, {% endfor %}*args, **kwargs):
         """
         :param request: An HttpRequest
```

Comment:
Can you please add a parameter description line here for `body` and `form_data` if it is available. Something like
```python
{% if info.body %}
:param body: dict A dictionary containing the parsed and validated body 
{% endif %}
{% if info.formdata %}
:param form_data: dict A dictionary containing form fields and their values. In the case where the form fields refer to uploaded files, the values will be instances of `django.core.files.uploadedfile.UploadedFile`
{% endif %}
```

Reply:
i thought of that, but formdata is in the request, so it won't ever be a parameter?

Reply:
oh wait, thats in stubs.py i see sorry you are right. I was thinking of views.py